src/settings/Settings.tsx

```diff
@@ -347,10 +347,8 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         displayName: _td("Show extensible event representation of events"),
         default: false,
     },
-    "feature_use_only_current_profiles": {
-        isFeature: true,
-        labsGroup: LabGroup.Rooms,
-        supportedLevels: LEVELS_FEATURE,
+    "useOnlyCurrentProfiles": {
+        supportedLevels: [SettingLevel.ACCOUNT],
```

Comment:
we should probably be using `LEVELS_ACCOUNT_SETTINGS` even if the UI doesn't expose it at the `device` level (so power users can enable it on their personal devices).

```suggestion
        supportedLevels: LEVELS_ACCOUNT_SETTINGS,
```